x-pack/platform/packages/shared/index-management/index_management_shared_types/src/types.ts

```diff
@@ -35,9 +36,15 @@ export type IndexManagementLocatorParams = SerializableRecord &
 
 export type IndexManagementLocator = LocatorPublic<IndexManagementLocatorParams>;
 
+export interface SearchIndicesAppMountParams {
```

Comment:
It seems like this should be described in terms of `ManagementAppMountParams`, hopefully its a superset of it.